lib/lambda/update/updatePackage.ts

```diff
@@ -0,0 +1,217 @@
+import { response } from "libs/handler-lib";
+import { APIGatewayEvent } from "aws-lambda";
+import { getPackage, getPackageChangelog } from "libs/api/package";
+import { produceMessage } from "libs/api/kafka";
+import { ItemResult } from "shared-types/opensearch/main";
+import { events } from "lib/packages/shared-types";
+import { z } from "zod";
+
+const sendDeleteMessage = async (topicName: string, packageId: string) => {
+  await produceMessage(
+    topicName,
+    packageId,
+    JSON.stringify({
+      id: packageId,
+      deleted: true,
+      isAdminChange: true,
+      adminChangeType: "delete",
+    }),
+  );
+};
+
+const sendUpdateValuesMessage = async (
+  topicName: string,
+  packageId: string,
+  updatedFields: object,
+  changeMadeText: string,
+  changeReason?: string,
+) => {
+  await produceMessage(
```

Comment:
Would it make sense to define `topicName` here and throw an Error when it's `undefined`? We wouldn't have to pass it into every function call

Reply:
Are you saying it should be a global variable in this file?

Reply:
We can do that as well. Or we can define it in the function

Reply:
Oh, so define it inside each function instead of the main handler function?

Reply:
If that makes more sense, sure. Global is also a good move, but I don't like the fact that the error isn't handled inside a function

Reply:
The error for an undefined topic name is handled in the main handler function, did you want to handle it in these smaller functions as well?

Reply:
Better to move that error handling into the smaller functions?